mailbox/event/json/src/main/scala/org/apache/james/event/json/DTOs.scala

```diff
@@ -76,56 +76,71 @@ object DTOs {
     def fromJava(javaMessageMetaData: JavaMessageMetaData): MessageMetaData = DTOs.MessageMetaData(
       javaMessageMetaData.getUid,
       javaMessageMetaData.getModSeq,
-      javaMessageMetaData.getFlags,
+      Flags.fromJavaFlags(javaMessageMetaData.getFlags),
       javaMessageMetaData.getSize,
       javaMessageMetaData.getInternalDate.toInstant,
       javaMessageMetaData.getMessageId)
   }
 
-  case class MessageMetaData(uid: MessageUid, modSeq: Long, flags: JavaMailFlags, size: Long, internalDate: Instant, messageId: MessageId) {
-    def toJava: JavaMessageMetaData = new JavaMessageMetaData(uid, modSeq, flags, size, Date.from(internalDate), messageId)
+  case class MessageMetaData(uid: MessageUid, modSeq: Long, flags: Flags, size: Long, internalDate: Instant, messageId: MessageId) {
+    def toJava: JavaMessageMetaData = new JavaMessageMetaData(uid, modSeq, Flags.toJavaFlags(flags), size, Date.from(internalDate), messageId)
   }
 
   sealed trait Flag
```

Comment:
this trait doesn't make sense anymore

Reply:
It do : see the Flags.toJava method relying on this.